components/SocialLinks/SocialLinks.tsx

```diff
@@ -15,8 +21,11 @@ function SocialLinks() {
         target="_blank"
         rel="noreferrer"
       >
-        <IconMail />
-        <span className="social-links__mail">
+        {pathname === AppRoute.Frontend ? <IconMailFrontend /> : <IconMail />}
```

Comment:
I think we need to take a different approach here, because we know that there will be more pages and there will be different icons.

We can look at the switch case

Reply:
Yes, I remember this, I wanted to implement it in the future.